recipes/exoplanet-core/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "exoplanet-core" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/exoplanet_core-{{ version }}.tar.gz
+  sha256: 33f25c7b77b3882cbf16a94636b791f68816b5d929fdb9c26d808bca93c7854e
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cmake
```

Comment:
`cmake` should be listed as a build dep, not host. (Matters for cross-compiling.)

Reply:
Done, thanks!